test/tests-gimletlet/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+edition = "2018"
+readme = "README.md"
+name = "tests-gemini-bu"
```

Comment:
Should this be `tests-gimletlet` (similarly the `[[bin]]` path below)?

Reply:
Too much copy-pasta, fixed!